src/main/resources/default/taglib/t/sidebarKeyMetrics.html.pasta

```diff
@@ -1,29 +1,30 @@
 <i:arg type="java.lang.Object" name="target"/>
 
-<i:pragma name="description">Renders a sindebar view of some key metrics for the given target as supplied by "MetricDashboard"</i:pragma>
+<i:pragma name="description">Renders a sindebar view of some key metrics for the given target as supplied by
+    "MetricDashboard"
+</i:pragma>
```

Comment:
This doesnt break Tagliatelle?

Reply:
![grafik](https://user-images.githubusercontent.com/875799/162427870-44752586-d8ff-4f7a-934c-1594cafd9623.png)
Nope...

Reply:
K, then it just looks strange :D

Reply:
K, then it just looks strange :D